index.html

```diff
@@ -0,0 +1,161 @@
+        </p>
+      </div>
+      <div class="date">
+        <p>2022.11.18(FRI) ~ 20(SUN)</p>
+      </div>
+      <div class="venue">
+        <p>
+          @Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        </p>
+      </div>
+    </div>
+
+  </section>
+
+  <section class="program" id="program">
+    <div class="top-tit">
+      <h2>Main Program</h2>
+    </div>
+    <div class="line"></div>
+    <div class="card-container">
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_01 1.png" alt="icon" /></div>
+        <div class="card-title">Workshops</div>
+        <div class="card-text">The Hackathon will offer Workshops and helpful mentor sessions to educate and
+          prepare participants</div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_02 1.png" alt="icon" /></div>
+        <div class="card-title">Network</div>
+        <div class="card-text">Connect with tech companies and other participants and so much more. </div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_03 1.png" alt="icon" /></div>
+        <div class="card-title">Challenge</div>
+        <div class="card-text">To test the technical competencies of young developers by offering them opportunities to
+          solve critical challenges</div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_02 1.png" alt="icon" /></div>
+        <div class="card-title">Innovation</div>
+        <div class="card-text">To boost a culture of innovation, idea sharing, effective collaboration and creativeness
+          driven by enthusiasm towards a shared vision. </div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_05 1.png" alt="icon" /></div>
+        <div class="card-title">Prizes</div>
+        <div class="card-text">Shared $3000 for top 3 teams</div>
+      </div>
+    </div>
+    <div class="show_all_program">
+      <a href="#" class="web">View all programs>></a>
+    </div>
+    <div>
+      <button class="register"> Register</button>
+    </div>
+  </section>
+
+  <section class="speakers" id="speakers">
+    <div class="top-speakers">
+      <h2>Featured Speakers</h3>
+    </div>
+    <div class="line"></div>
+    <div class="speaker-container"></div>
+  </section>
+
+  <section class="partners" id="partners">
+    <div class="top-speakers">
+      <h2>Partners</h3>
+    </div>
+    <div class="line"></div>
+    <div class="partners-container">
+
+      <img class="logo-pt" src="./pictures/tencent.webp">
+
+      <img class="logo-pt" src="./pictures/logo.png">
+
+      <img class="logo-pt" src="./pictures/logo2.png">
+
+      <img class="logo-pt" src="./pictures/logo3.png">
+
+      <img class="logo-pt" src="./pictures/ibm.jpg">
+
+  </section>
```

Comment:
- [ ] Kindly try to follow design guidelines as much as possible. For the partners section the images should be smaller and the section overall is much smaller in the guidelines.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198383858-e91ca5ce-cab4-489e-849a-aae96f2074ac.png)

## Design Guidelines:
![image](https://user-images.githubusercontent.com/50721479/198383888-73fc7294-9f3b-4550-8da6-31162b7bc47f.png)